### ISSUE TITLE
Rules with 2 input devices must have a transition to work

### ISSUE THREAD

**Issue by Tomasy** [mozilla-iot/gateway#1452]
When the rules contains more than one device as input it will not work before both devices have been not active.
Example:
If I add a rule saying `if DateTime minute is 5 and DateTime is dark, turn Fan on` The fan is not turned on the first day if the rule was created when it was dark. I had to wait until the next day.

**Comment by tomasy** [comment]
I think this is the same issue as reported in #1247

**Comment by tomasy** [comment]
Hi @mrstegeman ,
I am going to make a new release of DateTime adapter. Just to be seyre is this bug fixed so I can remove my workaround?

**Comment by mrstegeman** [comment]
No, still an issue, I believe. This issue was closed because it was a duplicate.
